src/commands/schema/push.ts

```diff
@@ -52,17 +43,23 @@ class PushSchemaCommand extends SchemaCommand {
         if (json.error) {
           this.error(json.error.message);
         }
-        let msg = "Accept and push changes?"
+        let message = "Accept and push changes?";
         if (json.diff) {
           this.log(`Proposed diff:\n`);
           this.log(json.diff);
         } else {
           this.log("No logical changes.");
-          msg = "Push file contents anyway?"
+          message = "Push file contents anyway?";
         }
-        if (await this.confirm(msg)) {
+        this.log(`Proposed diff:\n`);
+        this.log(json.diff);
```

Comment:
are we logging the diff twice here?

Reply:
oh yeah oops. let me go check, i think i messed up the rebase